### DISCUSSION TITLE
Renaming resource name from 'page'

### DISCUSSION THREAD

**Discussion by djsall** [Z3d0X/filament-fabricator#52] [Ideas]
As far as I can tell, the package doesn't provide a facade to rename the 'Pages' navigation label to something else, it only provides access to the group name and the navigation order. I could not find a way to set plural model name as-well, as it seems inside the source code, that the package either uses the plural name, or the default value.

This could also be solved with allowing to disable auto-registration of the menu item and allowing the users to register it for themselves, like on my screenshot, which results in double menu items, but it could work if I could disable the original item provided automatically by the package.

![Screenshot 2023-03-13 at 12 06 25](https://user-images.githubusercontent.com/4785203/224684436-f1b965ab-5156-4329-8b8d-2c187887b47f.png)
![Screenshot 2023-03-13 at 12 07 15](https://user-images.githubusercontent.com/4785203/224684595-18d71409-68c4-44f3-a58b-c5432c5a71d5.png)

**Comment by Z3d0X** [comment]
I think we could add it to the translation file?
https://github.com/Z3d0X/filament-fabricator/blob/main/resources/lang/en/page-resource.php

**Reply by djsall** [reply]
That would be great imo, also superbly easier to use than having to call functions in a service provider.
